build/build_edge.sh

```diff
@@ -40,6 +40,7 @@ install_recipes()
         echo "EXTERNALSRC = \"$XRT_REPO_DIR/src\"" >> $XRT_BB
         echo 'EXTERNALSRC_BUILD = "${WORKDIR}/build"' >> $XRT_BB
         echo 'PACKAGE_CLASSES = "package_rpm"' >> $XRT_BB
+        echo 'INSANE_SKIP_${PN}-dev += "dev-elf"' >> $XRT_BB
```

Comment:
#3176 has been merged. This line is not necessary.
BTW, #3187 do cleanup the libraries install path. It is final solution to fix the QA issue.

Reply:
@Himanshu-xilinx , As the original issue is fixed, please enable the check again.

Reply:
updated